app/src/main/java/com/mk/tasky/agenda/detail/components/model/ReminderTypes.kt

```diff
@@ -0,0 +1,9 @@
+package com.mk.tasky.agenda.detail.components.model
+
+sealed class ReminderTypes(val type: String) {
```

Comment:
Consider making this an enum class instead. That allows you to easily loop over all reminders by using `ReminderTypes.values()`

Reply:
Nice idea!